ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/CatalogController.java

```diff
@@ -254,17 +265,28 @@ private Map<String, Object> authConvertFromMetaToServer(
                     catalogName, CONFIG_TYPE_AUTH, AUTH_CONFIGS_KEY_KRB5.replace("\\.", "-"))));
         break;
       case AUTH_CONFIGS_VALUE_TYPE_AK_SK:
-        CatalogUtil.copyProperty(metaAuthConfig, serverAuthConfig, AUTH_CONFIGS_KEY_ACCESS_KEY);
-        CatalogUtil.copyProperty(metaAuthConfig, serverAuthConfig, AUTH_CONFIGS_KEY_SECRET_KEY);
+        CatalogUtil.copyProperty(
+            catalogMeta.getCatalogProperties(),
+            serverAuthConfig,
+            S3FileIOProperties.ACCESS_KEY_ID,
+            AUTH_CONFIGS_KEY_ACCESS_KEY);
+        catalogMeta.getCatalogProperties().remove(S3FileIOProperties.ACCESS_KEY_ID);
+        CatalogUtil.copyProperty(
+            catalogMeta.getCatalogProperties(),
+            serverAuthConfig,
+            S3FileIOProperties.SECRET_ACCESS_KEY,
+            AUTH_CONFIGS_KEY_SECRET_KEY);
+        catalogMeta.getCatalogProperties().remove(S3FileIOProperties.SECRET_ACCESS_KEY);
         break;
     }
 
     return serverAuthConfig;
   }
 
   private Map<String, Object> storageConvertFromMetaToServer(
```

Comment:
`pareStorageConfigsFromCatalogMeta` may be a proper method name.

Reply:
I rename it to `extractStorageConfigsFromCatalogMeta`